Readme.md

```diff
@@ -1,21 +1,27 @@
-WWTD: Travis simulator - faster + no more waiting for build emails.<br/>
-Reads your .travis.yml and runs what travis would run (via rvm/rbenv/chruby).
+# WWTD: Travis simulator [![Build Status](https://travis-ci.org/grosser/wwtd.svg?branch=master)](https://travis-ci.org/grosser/wwtd)
+
+Reads your `.travis.yml` and runs what Travis would run using [rvm], [rbenv], or [chruby].  No more waiting for build emails!
 
 ![Results](assets/results.png?raw=true)
 
-Install
-=======
+[rvm]: https://github.com/rvm/rvm
+[rbenv]: https://github.com/rbenv/rbenv
+[chruby]: https://github.com/postmodern/chruby
+
+
+## Install
 
-```Bash
-gem install wwtd
+```bash
+$ gem install wwtd
```

Comment:
sry, but I hate these `$` things because they block copy-pasting :D

Reply:
Gotcha!  The real motivation for adding these was the first example under "Usage."  I wanted to add a visible delineation between the `wwtd` command and `START gemfile: ...` output.  Should I just remove that output since the image at the top of the Readme already hints at it?

Reply:
yeah, just kill the output and add a line for just `wwtd # blah blah`